x/market/internal/types/params.go

```diff
@@ -30,16 +32,23 @@ var (
 	DefaultPoolRecoveryPeriod = core.BlocksPerDay                   // 14,400
 	DefaultMinSpread          = sdk.NewDecWithPrec(2, 2)            // 2%
 	DefaultTobinTax           = sdk.NewDecWithPrec(25, 4)           // 0.25%
+	DefaultTobinTaxList       = TobinTaxList{
+		{
+			Denom:   core.MicroMNTDenom,
+			TaxRate: sdk.NewDecWithPrec(2, 2), // 2%
+		},
+	}
 )
 
 var _ subspace.ParamSet = &Params{}
 
 // Params market parameters
 type Params struct {
-	PoolRecoveryPeriod int64   `json:"pool_recovery_period" yaml:"pool_recovery_period"`
-	BasePool           sdk.Dec `json:"base_pool" yaml:"base_pool"`
-	MinSpread          sdk.Dec `json:"min_spread" yaml:"min_spread"`
-	TobinTax           sdk.Dec `json:"tobin_tax" yaml:"tobin_tax"`
+	PoolRecoveryPeriod int64        `json:"pool_recovery_period" yaml:"pool_recovery_period"`
+	BasePool           sdk.Dec      `json:"base_pool" yaml:"base_pool"`
+	MinSpread          sdk.Dec      `json:"min_spread" yaml:"min_spread"`
+	TobinTax           sdk.Dec      `json:"tobin_tax" yaml:"tobin_tax"`
+	TobinTaxList       TobinTaxList `json:"tobin_tax_list" yaml:"tobin_tax_list"`
```

Comment:
IlliquidTobinTaxList // List of denoms that pay a higher tobin tax due to illiquidity